ipatests/prci_definitions/temp_commit.yaml

```diff
@@ -72,7 +72,34 @@ jobs:
       class: RunPytest
       args:
         build_url: '{fedora-latest/build_url}'
-        test_suite: test_integration/test_REPLACEME.py
+        test_suite: test_integration/test_otp.py
         template: *ci-master-latest
         timeout: 3600
-        topology: *master_1repl_1client
+        topology: *master_1repl
+
+  sssd-fedora/build:
+    requires: []
+    priority: 100
+    job:
+      class: Build
+      args:
+        copr: '@sssd/nightly'
+        git_repo: '{git_repo}'
+        git_refspec: '{git_refspec}'
+        template: *ci-master-latest
+        timeout: 1800
+        topology: *build
+
+  sssd-fedora/test_otp:
+    requires: [sssd-fedora/build]
+    priority: 50
+    job:
+      class: RunPytest
+      args:
+        build_url: '{sssd-fedora/build_url}'
+        update_packages: True
+        copr: '@sssd/nightly'
+        test_suite: test_integration/test_otp.py
+        template: *ci-master-latest
+        timeout: 3600
+        topology: *master_1repl
```

Comment:
Can you also add the following to your temp commit? This will also test f39.
```
  fedora-previous/build:
    requires: []
    priority: 100
    job:
      class: Build
      args:
        git_repo: '{git_repo}'
        git_refspec: '{git_refspec}'
        template: &ci-master-previous
          name: freeipa/ci-master-f39
          version: 0.0.1
        timeout: 1800
        topology: *build

  fedora-previous/test_otp:
    requires: [fedora-previous/build]
    priority: 50
    job:
      class: RunPytest
      args: 
        build_url: '{fedora-previous/build_url}'
        test_suite: test_integration/test_otp.py
        template: *ci-master-previous
        timeout: 3600
        topology: *master_1repl
```